js/data.js

```diff
@@ -6,3 +6,15 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+if (localStorage.getItem('data')) {
+  for (var key in data) {
+    var previousDataJSON = localStorage.getItem('data');
+    var previousData = JSON.parse(previousDataJSON);
+    data[key] = previousData[key];
+  }
```

Comment:
I would just change this to `data = JSON.parse(previousDataJSON)`

Reply:
xD. Okay, seems easier